lib/checks/isQuadratic.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const Node = require('../node');
+
+// TODO(ael)
+function isQuadratic(node) {
+  if (!Node.Type.isOperator(node) || node.op !== '+') {
+    return false;
+  }
+
+  if (node.args.length > 3) {
+    return false;
+  }
+
+  const secondDegreeTerms = node.args.filter(isSecondDegree);
+  const firstDegreeTerms = node.args.filter(isFirstDegree);
+  const constantTerms = node.args.filter(Node.Type.isConstant);
```

Comment:
what if there are other args that don't fall into one of these buckets? we should return false right? so I guess check that the length of each of these add up to the number of args